SofascoreAcademy/SofascoreAcademy/View/LeagueComponentView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LeagueComponentView.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 16.03.2024..
+//
+
+import UIKit
+import SofaAcademic
+
+class LeagueComponentView: BaseView {
+    
+    private let leagueHeaderView: LeagueHeaderView = LeagueHeaderView()
+    private let eventsStackView: UIStackView = UIStackView()
```

Comment:
```suggestion
    private let leagueHeaderView: LeagueHeaderView = .init()
    private let eventsStackView: UIStackView = .init()
```

Možeš tako i na drugim mjestima